src/app/course/course-question-snippet/course-question-snippet.component.scss

```diff
@@ -7,3 +7,15 @@
 .question-islands {
     @include island-grid();
 }
+
+.teamBar{
+    padding: 1rem;
+    border-radius: 8px;
+    background-color: var(--tui-base-02);
+    display: flex;
+    align-items: baseline;
```

Comment:
Let's use tailwind classes `flex`, `items-center`, `p-4`, and `rounded` instead of the CSS. we still need the background color in CSS tho.